pkgs/by-name/tr/tracexec/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tracexec";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "kxxt";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-HQTNYReJkfiqS05APtLAUf9dnnPYTzF0ratd5QKv+8w=";
+  };
+
+  checkFlags = [
+    "--skip=cli::test::log_mode_without_args_works" # `Permission denied` (maybe needs root?)
+    "--skip=tracer::test::tracer_emits_exec_event" # needs `/bin/true`
+  ];
+
+  cargoHash = "sha256-Ry7dwaoEAVWl04ug3MI1UX92+ynnnuwA3jfmA/6X+L8=";
+
+  meta = with lib; {
+    platforms = platforms.linux;
+    description = "A small utility for tracing execve{,at} and pre-exec behavior";
+    mainProgram = "tracexec";
+    homepage = "https://github.com/kxxt/tracexec";
+    license = with licenses; [ gpl2 ];
```

Comment:
I ship a third party licenses file (generated by cargo-about) alongside tracexec. It would be nice to also include it here if it fits into the workflow of Nix. My PKGBUILD at AUR can be taken as an example: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tracexec#n37 .

Reply:
@kxxt I think that does not work here, because this field is supposed to be a simple string (or list of strings) to facilitate automatic querying.

See https://nixos.org/manual/nixpkgs/stable/#var-meta-license

If there are multiple licenses I should mention in the list, please let me know.

I added now the copying of the license to `share/` like you do in the AUR.

Reply:
> @kxxt I think that does not work here, because this field is supposed to be a simple string (or list of strings) to facilitate automatic querying.
> 
> See https://nixos.org/manual/nixpkgs/stable/#var-meta-license
> 
> If there are multiple licenses I should mention in the list, please let me know.
> 
> I added now the copying of the license to `share/` like you do in the AUR.

Thanks for your explanation! LGTM.